build.ps1

```diff
@@ -0,0 +1,15 @@
+echo "Building the frontend..."
+cd frontend
+npm i
+npm run build
+cd ..
+echo "Copying frontend files..."
+mkdir target\web
+xcopy frontend\build target\web /s /e
```

Comment:
This should preferably be done in powershell with `Move-Item` to keep things consistent.